iceoryx_posh/source/runtime/ipc_interface_creator.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2019 by Robert Bosch GmbH. All rights reserved.
+// Copyright (c) 2020 by Apex.AI Inc. All rights reserved.
```

Comment:
wrong year

Reply:
See [here](https://github.com/eclipse-iceoryx/iceoryx/pull/538#discussion_r573644456)

Reply:
The class was called MqInterfaceCreator before, therefore you touched and changed the code!